linux_os/guide/services/ssh/ssh_server/sshd_limit_user_access/oval/shared.xml

```diff
@@ -0,0 +1,57 @@
+<def-group>
+  <definition class="compliance" id="{{{ rule_id }}}" version="1">
+    {{{ oval_metadata("One of the following parameters of the sshd configuration file is set:  AllowUsers, DenyUsers, AllowGroups, DenyGroups.") }}}
+    <criteria operator="OR">
+      <criterion test_ref="test_allow_user_is_configured" />
+      <criterion test_ref="test_allow_group_is_configured" />
+      <criterion test_ref="test_deny_user_is_configured" />
+      <criterion test_ref="test_deny_group_is_configured" />
+    </criteria>
+  </definition>
+
+  <ind:textfilecontent54_test check="all"
```

Comment:
Nitpick, but I would recommend to use the available line length and save some lines in the file. Doing that, we could also include space between the objects. I believe these minor changes would improve the readability. e.g.:
```
  <ind:textfilecontent54_test check="all" id="test_allow_user_is_configured" version="1"
    check_existence="only_one_exists" comment="Check if there is an AllowUsers entry">
    <ind:object object_ref="obj_allow_user"/>
  </ind:textfilecontent54_test>

  <ind:textfilecontent54_test check="all" id="test_allow_group_is_configured" version="1"
      check_existence="only_one_exists" comment="Check if there is an AllowGroups entry">
    <ind:object object_ref="obj_allow_group" />
  </ind:textfilecontent54_test>
```

Reply:
@dodys , do you plan to work on this optional recommendation? Otherwise it looks ready to be merged.